tweetapi/login.py

```diff
@@ -0,0 +1,65 @@
+import jwt
+from .serializer import UserSerializer
+from rest_framework.generics import CreateAPIView
+import datetime
+from django.contrib.auth.models import User
+from django.contrib.auth import login,logout,authenticate
+from rest_framework.views import APIView
+from rest_framework import status
+from rest_framework.response import Response
+
+
+class Login(CreateAPIView):
+    serializer_class = UserSerializer
+    def post(self, request, *args, **kwargs):
+        username = request.data.get("username")
+        password = request.data.get("password")
+
+        if not username or not password:
+            return Response({"error":"please fill all fields"}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
You can use a serializer here

Reply:
Alright sir, notes will make changes

Reply:
?

Reply:
this is outdated sir made those changes already